src/helpers.js

```diff
@@ -77,6 +78,8 @@ const isLinterLintCommandDefined = (editor: TextEditor) =>
     .findCommands({ target: atom.views.getView(editor) })
     .some(command => command.name === LINTER_LINT_COMMAND);
 
+const getDepPath = dep => path.join(__dirname, '../node_modules', dep);
```

Comment:
can you add a flow annotation for what `dep` is supposed to be (I assume `string` or something, right?)